1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,6 +1,42 @@
-function setAlarm() {}
+let isPaused = false;
+let timer;
+let mini;
+let seconds;
 
-// DO NOT EDIT BELOW HERE
+function setAlarm() {
+  let timeRemaining = document.getElementById("timeRemaining");
+  let alarmSet = document.getElementById("alarmSet").value * 60;
+  if (alarmSet > 0) {
+    timer = setInterval(() => {
+      if (!isPaused) {
+        alarmSet--;
+        mini = Math.floor(alarmSet / 60);
+        seconds = alarmSet % 60;
+        seconds = seconds < 10 ? `0${seconds}` : seconds;
+        mini = mini < 10 ? `0${mini}` : mini;
+        timeRemaining.textContent = `Time Remaining: ${mini}:${seconds}`;
```

Comment:
Whilst this is fine you could break out into a separate function which displays formatted text.  It's good to avoid functions with lots and lots of lines by building each function out of other functions, helper functions if you like.